io.catenax.failure_pattern/1.0.0/FailurePattern.ttl

```diff
@@ -0,0 +1,272 @@
+   samm:characteristic :BooleanInsideOperatorEnumerator ;
+   samm:exampleValue "equal" .
+
+:booleanValue a samm:Property ;
+   samm:preferredName "Boolean value"@en ;
+   samm:description "The value that is used within the boolean expression and with which it is to be compared"@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:timestampPropertyName a samm:Property ;
+   samm:preferredName "Timestamp property name"@en ;
+   samm:description "A property name of type timestamp from the model."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "productionDate" .
+
+:timestampInsideOperator a samm:Property ;
+   samm:preferredName "Timestamp inside operator"@en ;
+   samm:description "Describes the logical connector inside of a timestamp expression"@en ;
+   samm:characteristic :TimtstampInsideOperatorEnumeration ;
+   samm:exampleValue "after" .
+
+:timestampValue a samm:Property ;
+   samm:preferredName "Timestamp value"@en ;
+   samm:description "The value that is used within the timestamp expression and with which it is to be compared"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2024-01-01T00:00:00"^^xsd:dateTime .
+
+:numericPropertyName a samm:Property ;
+   samm:preferredName "Numeric property name"@en ;
+   samm:description "A property name of a numeric type from the model."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "repairMileage" .
+
+:numericInsideOperator a samm:Property ;
+   samm:preferredName "Numeric inside operator"@en ;
+   samm:description "Describes the logical connector inside of a numeric expression"@en ;
+   samm:characteristic :NumericInsideOperatorEnumeration ;
+   samm:exampleValue "higher" .
+
+:numericValue a samm:Property ;
+   samm:preferredName "Numeric value"@en ;
+   samm:description "The value that is used within the numeric expression and with which it is to be compared"@en ;
+   samm:characteristic :NumericValueCharacteristic ;
+   samm:exampleValue "100"^^xsd:double .
+
+:StringInsideOperatorEnumeration a samm-c:Enumeration ;
+   samm:preferredName "String inside operator enumeration"@en ;
+   samm:description "Provides a selection of several logical operators for string expressions"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "equal" "not_equal" "like" ) .
+
+:LinkToAspectModel a samm:Characteristic ;
+   samm:preferredName "Link to aspect model"@en ;
+   samm:description "This characteristic groups all necessary properties to refer to an existing Catena-X aspect model."@en ;
+   samm:dataType :AspectModel .
+
+:BooleanInsideOperatorEnumerator a samm-c:Enumeration ;
+   samm:preferredName "Boolean inside operator enumeration"@en ;
+   samm:description "Provides a selection of several logical operators for boolean expressions"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "equal" "not_equal" ) .
+
+:TimtstampInsideOperatorEnumeration a samm-c:Enumeration ;
```

Comment:
typo in `TimtstampInsideOperatorEnumeration`

Reply:
fixed typo